src/parser/jobs/sam/modules/Meikyo.js

```diff
@@ -0,0 +1,90 @@
+import React, {Fragment} from 'react'
+//import {Icon, Message} from 'semantic-ui-react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const MEIKYO_GCDS = {
+
+	//These actions are bad to use under meikyo
```

Comment:
Would be a bit cleaner to implement this as a whitelist or blacklist (whichever is easier to deal with), and handle the counting aspect within the module. Using 0/1 keys in place of `++` leaves room for logic mistakes